Screenbox/App.xaml

```diff
@@ -18,15 +18,12 @@
                     <!--  Brushes  -->
                     <ResourceDictionary.ThemeDictionaries>
                         <ResourceDictionary x:Key="Default">
-                            <StaticResource x:Key="ListViewAltItemBackgroundBrush" ResourceKey="CardBackgroundFillColorDefaultBrush" />
                             <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorLight2" />
                         </ResourceDictionary>
                         <ResourceDictionary x:Key="Light">
-                            <StaticResource x:Key="ListViewAltItemBackgroundBrush" ResourceKey="CardBackgroundFillColorDefaultBrush" />
                             <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorDark1" />
                         </ResourceDictionary>
                         <ResourceDictionary x:Key="HighContrast">
-                            <StaticResource x:Key="ListViewAltItemBackgroundBrush" ResourceKey="SystemColorButtonFaceColorBrush" />
```

Comment:
Doesn't removing the last resource break HighContrast?

Reply:
It shouldn't, the CardBackgroundFillColorDefaultBrush is the "same". IIRC it's ButtonFace (currently) vs WindowColor.

Reply:
Confirmed